src/task1-1.js

```diff
@@ -1,18 +1,22 @@
-const readline = require("readline");
+const readline = require('readline');
+
 const rl = readline.createInterface({
   input: process.stdin,
-  output: process.stdout
+  output: process.stdout,
 });
 
-askQuestion();
-
 function reverse(str) {
-  return str.split("").reverse().join("")
+  return str
+    .split('')
+    .reverse()
+    .join('');
 }
 
 function askQuestion() {
-  rl.question("Type input: \n", input => {
+  rl.question('Type input: \n', (input) => {
```

Comment:
In output you;'re getting these lines in 1 iteration:
```
<input>
<input>
<output>
```
Try to think how you can remove the middle `<input>`. Note: good idea to use `process.stdin.on`

Reply:
I noticed this and couldn't work out why it was happening. The readline package was a little mysterious, so I took your note and updated to process.stdin.on... I didn't realize that 'data' would submit itself entirely as a chunk from the CLI.